hazelcast-jet-core/src/main/java/com/hazelcast/jet/aggregate/AggregateOperations.java

```diff
@@ -416,12 +482,12 @@ private AggregateOperations() {
      * See also {@link #filtering filtering()} and {@link #flatMapping
      * flatMapping()}.
      *
+     * @param mapFn the function to apply to input items
```

Comment:
Extend the comment in the code sample:
// calculate the set of surnames from the stream of Persons